### PR TITLE
Form CSS improvements. 

### PR DESCRIPTION
**Pull request by kevinroast**
Left-to-right field row rendering. Minor font size improvements for Alfresco Share consistency.

Layout changes only - no functional modifications.

### CODE REVIEW

**Comment by draperd** [comment]
I've checked out the latest changes on your branch and ran the full test suite, I'm getting the following error:

Text Box Tests
- Check validation error message
  "Validation error message should be displayed"

(FYI... You can target specific test suites by updating the Suites.js file - there's a section at the top we use for individual tests, just switch which version of "baseFunctionalSuites" has the "x" in front of it and set the the unit test as the entry in the array).

**Comment by draperd** [comment]
Also, there's still an issue with the validation icon being to the left on page forms (when it used to be on the right). I'm not sure if we want to worry about that or not.

**Comment by kevinroast** [comment]
>I've checked out the latest changes on your branch and ran the full test suite, I'm getting the following error:
>Text Box Tests
>Check validation error message "Validation error message should be displayed"

Because of course the test checks for a specific CSS class, which is no longer present because now the Form CSS is only applied in dialog. I've switched it back. Perhaps the test can be made less brittle? I've had to change it twice now and each time it has not pointed out an error at all.

**Comment by kevinroast** [comment]
>Also, there's still an issue with the validation icon being to the left on page forms (when it used to be on the right). I'm not sure if we want to worry about that or not.
That's not an issue.

File: aikau/src/main/resources/alfresco/forms/controls/css/BaseFormControl.css [hunk 1]
```diff
@@ -1,37 +1,31 @@
...
 }
 
-.alfresco-forms-controls-BaseFormControl > .title-row > label {
-   font-size: @normal-font-size;
-   line-height: ceil(@normal-font-size/@standard-line-height) * @standard-line-height;
-   margin-bottom: 3px;
-   display: inline-block;
-}
-
-.alfresco-forms-controls-BaseFormControl.hiddenlabel > .title-row {
```
Comment (draperd) [inline aikau/src/main/resources/alfresco/forms/controls/css/BaseFormControl.css]: You've removed the .hiddenLabel class but you've not removed all the references to it, e.g. from PdfJs.js, InlineEditProperty.js, InlineEditSelect.js, Tags.js. If we're removing the class we should probably also be removing references to it.

**Comment by kevinroast** [comment]
I haven't removed it... it's just moved down the file..!
+.alfresco-forms-controls-BaseFormControl.hiddenlabel > .title-row {
+   position: absolute;
+   left: -999em;
+   height: 0;
+}

**Pull request merged**